docs/README.md

```diff
@@ -19,417 +17,498 @@
+## Events
+Events are specified in the `when` tag like this:
 
-### Staleness
+```yml
+- when: pull_request.opened
+```
 
-Detect stale issues and pull requests. Notify authors and collaborators by leaving a comment. Staleness is defined through [configuration](#configuration).
+Multiple events for a recipe are declared comma delimited like this:
 
-## Vision
+```yml
+- when: pull_request.opened, issues.opened
+```
 
-The Mergeable vision is to make the software development effort efficient so that everyone can focus on building software.  
+Events supported for pull requests are as follows:
+`pull_request.opened`, `pull_request.edited`, `pull_request_review.submitted`, `pull_request_review.edited`, `pull_request_review.dismissed`, `pull_request.labeled`, `pull_request.unlabeled`, `pull_request.milestoned`, `pull_request.demilestoned`, `pull_request.assigned`, `pull_request.unassigned`, `pull_request.synchronize`,
 
-There are several areas in which we wish to automate for efficiency: consistency, workflow,  quality and statistics. The basic features for these areas are as follows:
+And for issues:
+`issues.opened`, `issues.edited`, `issues.labeled`, `issues.unlabeled`, `issues.milestoned`, `issues.demilestoned`, `issues.assigned`, `issues.unassigned`, `issues.synchronize`
 
-### Consistency
+>  ☝ **NOTE:** More details about events can be found on the [GitHub events page](https://developer.github.com/v3/activity/events).
 
-- *Pull Request* validation for standards based on configured rulesets. ![completed](https://img.shields.io/badge/Status-completed-green.svg)
+For convenience, wildcards can be used: `pull_request.*`, `issues.*`, `pull_request_review.*`
 
-- Notify by creating a comment in *Issues* that do not adhere to configured ruleset. ![completed](https://img.shields.io/badge/Status-completed-green.svg)
 
-- Ruleset across Repos. As a way to enforce and encourage standards in an organization or guide team members on the organizations' engineering best practises.
+> ☝ **NOTE:**  Each `validator` and `action` declares it's own supported events. Read the [validator](#validators) and [action](#actions) sections to find out which events are supported respectively.
 
-- Repo(s) audit. Scan repo(s) for standards configured in rulesets including existance OWNERS file, or .github contains TEMPLATES. Notify through creation of an issue in the repo(s).
+## Validators
 
-### Workflow
+### approvals
 
-- Kanban WIP limits. Limit the number of open Pull Requests by author.
+```yml
+- do: approvals
+  min:
+    count: 2 # Number of minimum reviewers. In this case 2.
+    message: 'Custom message...'
+  required:
+    reviewers: [ user1, user2 ]   # list of github usernames required to review
+    owners: true # accepts boolean. When true,  the file .github/CODEOWNER is read and owners made required reviewers
```

Comment:
I think we should explicitly say that this won't work with the public mergeable version right now because we don't have permission to read `.github/CODEOWNER` file

Reply:
It works for public repos and I think we should set the permissions shortly after we turn flex on. I think the time span of that is small enough to not mention it. Otherwise we'd have to come back to update it. For now I view it as a bug.